src/main/java/dk/kb/present/View.java

```diff
@@ -154,6 +154,7 @@ private Map<String,String> createBasicMetadataMap(DsRecordDto record) {
         final Map<String, String> metadata = new HashMap<>();
         metadata.put("recordID", record.getId());
         metadata.put("origin", origin);
+        metadata.put("conditionsOfAccess", "placeholderCondition");
```

Comment:
Nice placeholder. Maybe add a TODO to show that it is intentional to have it here until circumstances change?